lib/phoenix_html/link.ex

```diff
@@ -67,12 +67,12 @@ defmodule Phoenix.HTML.Link do
     raise ArgumentError, "link/2 requires a keyword list as second argument"
   end
 
-  def link(text, opts) do
-    {to, opts} = Keyword.pop(opts, :to)
+  def link(text, original_opts) do
+    {to, opts} = Keyword.pop(original_opts, :to)
     {method, opts} = Keyword.pop(opts, :method, :get)
 
     unless to do
-      raise ArgumentError, "option :to is required in link/2"
+      raise ArgumentError, "option :to is required in link/2 and its value cannot be nil, received: #{inspect original_opts}"
```

Comment:
I don't think we need `original_opts`, this should be enough:

    raise ArgumentError, "expected non-nil value for :to in link/2, got: #{inspect to}"

Can you please amend? Thank you!